Build.fsx

```diff
@@ -19,7 +19,9 @@ let build target configuration =
     let properties = [ "Configuration", configuration ] @ keyFile
 
     solutionsToBuild
-    |> MSBuild "" target properties
+    |> Seq.iter (fun s -> build (fun p -> { p with Verbosity = Some(Minimal)
```

Comment:
I did that intentionally, as it was impossible to track review the build log on CI - browser hung after the pressure of number of build entries.
However, later (in migration to .NET Core branch I'm currently working on) I found that we perform a lot of redundant rebuilds, so probably we could revert this setting back. Will update soon.